safe-global/safe-wallet-monorepo
# fix: React props errors

## What it solves

Fixes several React errors in the console

## How to test it

1. Create new Safe and open the network selector
2. Observe no errors in the console
3. Create new transaction and open the address book input and nonce form
4. Observe no errors in the console
5. Open the app and observe no "empty href is passed to prop" error
6. Go to the Settings page
7. Observe no ref error for Tabs and Tabs are still keyboard navigatable

## Checklist
